node-graph/gsvg-renderer/src/renderer.rs

```diff
@@ -1036,70 +1147,188 @@ impl Render for Table<Vector> {
 					scene.stroke(&outline_stroke, kurbo::Affine::new(element_transform.to_cols_array()), outline_color, None, &path);
 				}
 				_ => {
-					if use_layer {
-						let mut element = row.element.clone();
-						element.style.clear_stroke();
-						element.style.set_fill(Fill::solid(Color::BLACK));
-
-						let vector_table = Table::new_from_row(TableRow {
-							element,
-							alpha_blending: *row.alpha_blending,
-							transform: *row.transform,
-							source_node_id: None,
-						});
-
-						let bounds = row.element.bounding_box_with_transform(multiplied_transform).unwrap_or(layer_bounds);
-						let weight = row.element.style.stroke().as_ref().map_or(0., Stroke::effective_width);
-						let quad = Quad::from_box(bounds).inflate(weight * max_scale(applied_stroke_transform));
-						let bounds = quad.bounding_box();
-						let rect = kurbo::Rect::new(bounds[0].x, bounds[0].y, bounds[1].x, bounds[1].y);
-
-						let compose = if row.element.style.stroke().is_some_and(|x| x.align == StrokeAlign::Outside) {
-							peniko::Compose::SrcOut
-						} else {
-							peniko::Compose::SrcIn
-						};
-
-						if wants_stroke_below {
-							scene.push_layer(peniko::Mix::Normal, 1., kurbo::Affine::IDENTITY, &rect);
-							vector_table.render_to_vello(scene, parent_transform, _context, &render_params.for_alignment(applied_stroke_transform));
-							scene.push_layer(peniko::BlendMode::new(peniko::Mix::Normal, compose), 1., kurbo::Affine::IDENTITY, &rect);
+					if is_branching {
```

Comment:
I don't like having the two rendering paths separate. For example in the image below, I have appended a segment to the bottom, which ended up removing the red fill from the top shape. I don't think that the visual outcome should depend on having a ≥3 connected point anywhere in the entire network (spooky action at a distance).

<img width="719" height="486" alt="two vertical stacked V shapes initially filled with red, but then empty after a random segment is appended far away" src="https://github.com/user-attachments/assets/cb3dad37-642d-49d8-aa45-b6e5428bcb50" />